production/helm/loki/values.yaml

```diff
@@ -1796,6 +1796,13 @@ ingester:
   readinessProbe: {}
   # -- liveness probe settings for ingester pods. If empty use `loki.livenessProbe`
   livenessProbe: {}
+  # -- UpdateStrategy for the ingester StatefulSets.
+  updateStrategy:
+    # -- One of  'OnDelete' or 'RollingUpdate'
+    type: RollingUpdate
+    # -- Optional for updateStrategy.type=RollingUpdate. See [Partitioned rolling updates](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions) in the StatefulSet docs for details.
+    # rollingUpdate:
```

Comment:
If you specify a `spec.updateStrategy.rollingUpdate` block in values.yaml, you have to null it out if you want to set `spec.updateStrategy.type=OnDelete`. But I think `partition: 0` is a no-op in any case.